lib/views/after_auth_screens/chat/chat_list_screen.dart

```diff
@@ -1,20 +1,15 @@
-// ignore_for_file: talawa_api_doc
-// ignore_for_file: talawa_good_doc_comments
-
 import 'package:flutter/material.dart';
 import 'package:talawa/views/after_auth_screens/chat/direct_chats.dart';
-import 'package:talawa/views/after_auth_screens/chat/event_chats.dart';
 import 'package:talawa/views/after_auth_screens/chat/select_contact.dart';
 
-/// ChatPage returns a stateless widget for current user Chat Page which renders
-/// the list of all the users that the current user has chat with.
+/// A stateless widget for the current user's Chat Page, rendering the list of users with whom the current user has chatted.
```

Comment:
Any doc or code line is supposed to be in the range of 80 characters or so. If it gets longer, you are supposed to turn to multi-line, as discussed in another PR.

Reply:
Sorry for the delay, Please check now.